client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -463,6 +472,7 @@ export class List extends React.Component {
 
 		return [
 			<QuerySitePurchases key="query-purchases" siteId={ selectedSite.ID } />,
```

Comment:
Could we bring back the active features as part of this purchases query? I'm curious if we could get away with a single API call.

Reply:
sorry, are you suggesting using the same query component for both requests?

Reply:
I was wondering if that would be possible. We're interested in the subscriptions that have been purchased for the site, and this retrieves that from the API. I was wondering if we could extend it with the features data.

It's probably a fairly big change to the approach, and I'm not sure if it's possible or makes sense, so we can leave it and come back to it later if necessary.